site/Dance/index.php

```diff
@@ -153,17 +75,9 @@
       </div>
     </div>
 
-    <div class="header-box">
-      <img class="header-box-image" src="../assets/images/placeholder-image-rectangle.png">
-      <div class="header-box-title">
-        <h1>Dance</h1>
-      </div>
-    </div>
-    <div>
-      <p class="header-text">Jodie McSally Dance Program, Class of 21</p>
-    </div>
-
+    <?php template("hero.php", array('headertext' => 'Dance')) ?>
 
+<?php template("hero.php", array('headertext2' => 'Jodie McSally Dance Program, Class of 21')) ?>
```

Comment:
I think you can just use 1 variable for the `hero.php` file, so you should make this `headertext` instead of `headertext2` I think.